routes/addImgSlider.routes.js

```diff
@@ -0,0 +1,47 @@
+const { Router } = require('express') 
+const fs = require('fs')
+const path = require('path') 
+const router = Router()
+const imgModel = require('../models/slider')
+const multer = require('multer')
+
+const storage = multer.diskStorage({
+    destination: (req, file, cb) => {
+        cb(null, 'uploads')
+    },
+    filename: (req, file, cb) => {
+        cb(null, file.fieldname + '-' + req.body.name + '-' + Date.now())
+    }
+})
+ 
+const upload = multer({ storage: storage })
+  
+router.get('/', (req, res) => {
+    imgModel.find({}, (err, items) => {
+        if (err) {
+            console.log(err)
+            res.status(500).send('An error occurred', err)
+        } else {
+            res.render('imagesPage', { items: items })
+        }
+    })
+})
+
+router.post('/addimage', upload.single('image'), (req, res, next) => {
```

Comment:
`add-image`. But would be better `create-image`

Reply:
fixed